serving/src/main/java/feast/serving/configuration/ServingServiceConfig.java

```diff
@@ -79,9 +83,9 @@ public ServingService servingService(
 
     switch (store.getType()) {
       case REDIS:
-        servingService =
-            new RedisServingService(
-                storeConfiguration.getServingRedisConnection(), specService, tracer);
+        OnlineRetriever redisRetriever =
```

Comment:
Just flagging that we should probably have two "store types" that a user can define, so that we dont have one case statement that deals with both batch and online stores. Right now the user has to know what a store will be used for since we will select online/batch for them behind the scenes. A dual purpose store might be a problem.

Reply:
We can refactor this in another PR.